src/pages/Login/components/Login/LoginComponent.tsx

```diff
@@ -21,7 +21,7 @@ const LoginComponent = () => {
 
   const { setLogin } = useAuthUserStore()
   const navigate = useNavigate()
-  const { isModalToggle: isShowModal, showModal, closeModal } = useModal()
+  const { isShowModal: isShowModal, showModal, closeModal } = useModal()
```

Comment:
오른쪽에 타입 별칭 부분은 삭제하셔도 될 것 같습니다!
```suggestion
  const { isShowModal, showModal, closeModal } = useModal()
```

Reply:
엥 요건 제께 아닌데 왜들어갔을까요 ㅋㅋㅋㅋ

Reply:
이게 왜 여기에.. ㅋㅋㅋㅋ